File: js/acceso.js
let usuario_correcto = "David"
let contraseña_correcta = "12345"
let usuario_correcto1 = "Keiner"
let contraseña_correcta1 = "54321"

let usuario = "Keiner"
let contraseña = "54321"

if (usuario == usuario_correcto && contraseña == contraseña_correcta) {
    console.log("Bienvenido")
    console.log("Su rol es aprendiz")
}
else if (usuario = usuario_correcto1 && contraseña == contraseña_correcta1) {
    console.log("Bienvenido")
    console.log("Su rol es ADMIN")
}
else {
    console.log("Error de ingreso")
}

File: py/acceso.py
usuario_correcto1 = "David"
contraseña_correcta1 = "12345"

usuario_correcto2 = "Keiner"
contraseña_correcta2 = "654321"

usuario = input("Ingresa el usuario: ")
contraseña = input("Ingrese la contraseña: ")

if usuario == usuario_correcto1 and contraseña == contraseña_correcta1:
    print("Bienvenido")
    print("Tu rol es aprendiz")
elif usuario == usuario_correcto2 and contraseña == contraseña_correcta2:
    print("Bienbenido")
    print("Tu rol es ADMIN")
else:
    print("Error de ingreso")
    